### PR TITLE
refactored models/colors.ts

### PR DESCRIPTION
**Pull request by tuansydau**
## What
* deleted models/colors.ts
* added models/color.ts which specifies the allowed colors for landing components
* added 3 helper functions to convert from the color type to stroke/bg/drop-shadow
* deleted any references to hex color enums in favor of tailwind classes

## Testing
* Landing pages are identical between sealcred.xyz and localhost

### CODE REVIEW

File: src/icons/OrbBox.tsx [hunk 1]
```diff
@@ -1,34 +1,34 @@
...
-    dropShadow(shadowColor),
     opacity(shadow ? 'opacity-100' : 'opacity-90'),
     transitionProperty('transition-all')
   )
 const fillPath = fill('fill-primary-dark')
 
 interface OrbBoxProps {
-  color: Colors
+  color: Color
   shadow?: boolean
```
Comment (backmeupplz) [inline src/icons/OrbBox.tsx:19]: This seems like a misnomer. Should it be named `isShadowOpaque`?

File: src/components/landing/OrbsInBoxes.tsx [hunk 2]
```diff
@@ -33,11 +34,11 @@ const orbBox = (margins?: TMargin) =>
     space('space-y-2'),
     margin(margins)
   )
-const highlightedWrapper = (color: Colors) =>
+const highlightedWrapper = (color: Color) =>
   classnames(
     padding('px-2', 'py-1'),
     borderRadius('rounded-full'),
-    backgroundColor(colorToTailwindBg(color)),
+    backgroundColor(colorToTailwindBackground(color)),
```
Comment (backmeupplz) [inline src/components/landing/OrbsInBoxes.tsx:41]: Isn't it already wrapped in `backgroundColor`?

File: src/components/landing/OrbsInBoxes.tsx [hunk 3]
```diff
@@ -48,29 +49,29 @@ export default function () {
...
+          <ZkSphere color="accent" animated />
         </div>
-        <div className={highlightedWrapper(Colors.accent)}>
+        <div className={highlightedWrapper('accent')}>
           <HighlightedText center>Wallet 02</HighlightedText>
         </div>
       </div>
       <div className={orbBox('mt-11')}>
-        <OrbBox color={Colors.secondary} shadow={animNotStarted} />
+        <OrbBox color={'secondary'} shadow={animNotStarted} />
```
Comment (backmeupplz) [inline src/components/landing/OrbsInBoxes.tsx:70]: Use `""`

File: src/components/landing/OrbsInBoxes.tsx [hunk 4]
```diff
@@ -48,29 +49,29 @@ export default function () {
...
         <div className={position('absolute')}>
-          <ZkSphere color={Colors.secondary} animated />
+          <ZkSphere color={'secondary'} animated />
```
Comment (backmeupplz) [inline src/components/landing/OrbsInBoxes.tsx:72]: Use `""`

**Review by backmeupplz — CHANGES_REQUESTED** [review]

File: src/components/landing/OrbsInBoxes.tsx (hunk 2 above)
Comment (tuansydau) [reply src/components/landing/OrbsInBoxes.tsx:41]: This changes the color from 'accent' to 'bg-accent', and then backgroundColor does the type check - should this work differently?

File: src/components/landing/OrbsInBoxes.tsx (hunk 4 above)
Comment (tuansydau) [reply src/components/landing/OrbsInBoxes.tsx]: fixed

File: src/components/landing/OrbsInBoxes.tsx (hunk 3 above)
Comment (tuansydau) [reply src/components/landing/OrbsInBoxes.tsx]: fixed

File: src/icons/OrbBox.tsx (hunk 1 above)
Comment (tuansydau) [reply src/icons/OrbBox.tsx]: renamed it

**Review by backmeupplz — APPROVED** [review]

**Pull request merged**